graphql/mutation_resolver.go

```diff
@@ -883,16 +883,12 @@ func (r *mutationResolver) RemoveVolume(ctx context.Context, volumeID string) (b
 }
 
 // UpdateSpawnHostStatus is the resolver for the updateSpawnHostStatus field.
-func (r *mutationResolver) UpdateSpawnHostStatus(ctx context.Context, hostID *string, action *SpawnHostStatusActions, updateSpawnHostStatusInput *UpdateSpawnHostStatusInput) (*restModel.APIHost, error) {
-	shouldKeepOff := false
-	// TODO: Use input object throughout resolver once deprecated fields are removed
-	if updateSpawnHostStatusInput != nil {
-		hostID = &updateSpawnHostStatusInput.HostID
-		action = &updateSpawnHostStatusInput.Action
-		shouldKeepOff = utility.FromBoolPtr(updateSpawnHostStatusInput.ShouldKeepOff)
+func (r *mutationResolver) UpdateSpawnHostStatus(ctx context.Context, updateSpawnHostStatusInput UpdateSpawnHostStatusInput) (*restModel.APIHost, error) {
+	hostID := updateSpawnHostStatusInput.HostID
+	action := updateSpawnHostStatusInput.Action
+	shouldKeepOff := utility.FromBoolPtr(updateSpawnHostStatusInput.ShouldKeepOff)
 
-	}
-	h, err := host.FindOneByIdOrTag(ctx, *hostID)
+	h, err := host.FindOneByIdOrTag(ctx, hostID)
```

Comment:
not really related to the PR but I noticed we don't check if `h` is `nil`. Just taking a quick look at the function it seems possible that it would return (`nil`, `nil`). Should we check if `h` is `nil`?